### PR TITLE
Include the module version and some code formatting

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,17 +4,18 @@ Terraform module which creates EC2 instance(s) on AWS.
...
-* [EC2 instance](https://www.terraform.io/docs/providers/aws/r/instance.html) 
+* [EC2 instance](https://www.terraform.io/docs/providers/aws/r/instance.html)
 
 ## Usage
 
 ```hcl
 module "ec2_cluster" {
-  source = "terraform-aws-modules/ec2-instance/aws"
+  source                 = "terraform-aws-modules/ec2-instance/aws"
+  version                = "1.12.0"
```
Comment (chenrui333) [inline README.md]: Just want to explicitly specify the version of the module.

**Comment by chenrui333** [comment]
Can I get a review on this?

**Comment by antonbabenko** [comment]
Thanks, @chenrui333 !

I was (and still is) pretty busy to make reviews and releases faster.

**Comment by antonbabenko** [comment]
[v1.13.0](https://github.com/terraform-aws-modules/terraform-aws-ec2-instance/releases/v1.13.0) has been released.

**Pull request merged**